<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OGGExtractorMainWindow</class>
 <widget class="QMainWindow" name="OGGExtractorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OGG Extractor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc/resources.qrc">
    <normaloff>:/OGGExtractor/application.svg</normaloff>:/OGGExtractor/application.svg</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0,0,1,0,0,0,0">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data files to scan for OGG sound files:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="m_containersList">
        <property name="toolTip">
         <string>Data files to be scanned for OGG files.</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>OGG files found:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_streamsCount">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_addFile">
        <property name="toolTip">
         <string>Add data files to the list to be scanned for OGG files.</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_removeFile">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Remove data files from the list to be scanned for OGG files.</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="m_filesTableLabel">
      <property name="text">
       <string>OGG files found:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="m_filesTable">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>Found OGG files.</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="m_previous">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Go to the previous page.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rsc/resources.qrc">
          <normaloff>:/OGGExtractor/previous.svg</normaloff>:/OGGExtractor/previous.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="m_pageCount">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Current page.</string>
        </property>
        <property name="text">
         <string>[Empty]</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="m_next">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Go to the next page.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="rsc/resources.qrc">
          <normaloff>:/OGGExtractor/next.svg</normaloff>:/OGGExtractor/next.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="m_size">
          <property name="toolTip">
           <string>Don't show OGG files smaller than the given size.</string>
          </property>
          <property name="text">
           <string>Do not show files smaller than</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="m_minimumTime">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> seconds</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="m_time">
          <property name="text">
           <string>Do not show files with a duration less than</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_minimumSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Size in Kilobytes.</string>
          </property>
          <property name="suffix">
           <string> Kb</string>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>999999999</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Playing Volume</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QSlider" name="m_volumeSlider">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_volumeLabel">
            <property name="text">
             <string>100 %</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QProgressBar" name="m_progress">
        <property name="toolTip">
         <string>Current process progress.</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_cancel">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Cancel current process.</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="rsc/resources.qrc">
          <normaloff>:/OGGExtractor/cancel.svg</normaloff>:/OGGExtractor/cancel.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="m_quit">
        <property name="toolTip">
         <string>Quit the application. </string>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_about">
        <property name="toolTip">
         <string>About OGG Extractor.</string>
        </property>
        <property name="text">
         <string>About...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="m_scan">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Scan the data files in the list for OGG files. </string>
        </property>
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_extract">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Extract selected OGG files to disk.</string>
        </property>
        <property name="text">
         <string>Extract</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="rsc/resources.qrc"/>
 </resources>
 <connections/>
</ui>
